### PR TITLE
[Javascript]: Add nanoexpress pro

### PR DESCRIPTION
**Pull request by dalisoft**

### CODE REVIEW

**Comment by dalisoft** [comment]
@waghanza Can we somehow add `EXPERIMENTAL_FASTCALL=1` env before starting all `nanoexpress` server?

**Comment by dalisoft** [comment]
@waghanza Is this [commit](https://github.com/the-benchmarker/web-frameworks/pull/2327/commits/1f4c698581969a545f5c954c90e11c0750411f04) adds ENV variables correctly?

**Comment by waghanza** [comment]
@dalisoft nop. I updated your `PR`.

Feel free to ask me anything about this project, at least I will write a doc in a far future ...

In this case, you need a mix

+ add in `Dockerfile`
+ add in `config.yaml`

**Comment by dalisoft** [comment]
Thanks

File: javascript/nanoexpress-pro/config.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+framework:
+  website: nanoexpress/pro
+  version: 1.10
```
Comment (waghanza) [inline javascript/nanoexpress-pro/config.yaml]: could you surrond by quotes ?

File: javascript/nanoexpress-pro/config.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+framework:
+  website: nanoexpress/pro
```
Comment (waghanza) [inline javascript/nanoexpress-pro/config.yaml]: what is the website for `nanoexpress-pro` ?

File: javascript/nanoexpress-pro/package.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "nanoexpress-pro": "^1.10.10"
```
Comment (waghanza) [inline javascript/nanoexpress-pro/package.json]: could you use `~` instead of ` ^`

File: javascript/nanoexpress-pro/config.yaml (hunk 2 above)
Comment (dalisoft) [reply javascript/nanoexpress-pro/config.yaml]: Oh sorry, it's github repo

File: javascript/nanoexpress-pro/package.json (hunk 3 above)
Comment (dalisoft) [reply javascript/nanoexpress-pro/package.json]: done

File: javascript/nanoexpress-pro/config.yaml (hunk 1 above)
Comment (dalisoft) [reply javascript/nanoexpress-pro/config.yaml]: required for GitHub repo too?

Comment (waghanza) [reply javascript/nanoexpress-pro/config.yaml]: ah, then you must use `github` keyword

Comment (dalisoft) [reply javascript/nanoexpress-pro/config.yaml]: I used github keyword in latest commit

**Pull request merged**